docs/notebooks/jax-101/01-jax-basics.md

```diff
@@ -0,0 +1,375 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.0
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
++++ {"id": "6_117sy0CGEU"}
+
+# JAX as accelerated Numpy
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jax/blob/master/docs/notebooks/jax-101/01-jax-basics.ipynb)
+
+In this first colab you will learn the very fundamentals of JAX.
```

Comment:
Suggestion: 
```suggestion
In this first Colab you will learn the very fundamentals of JAX.
```

Is it first as in first 101 Colab? In the TOC, it appears to be below the famous Quickstart and Thinking in JAX, so it's technically third 🤔 Maybe it's best to say "first JAX 101 Colab"? I dunno

Reply:
Suggestion (a follow-up to the previous comment):

```suggestion
In this first JAX 101 Colab, you will learn the very fundamentals of JAX.
```

🤷